homeassistant/components/twinkly/light.py

```diff
@@ -268,8 +321,20 @@ async def async_update(self) -> None:
                 if key not in HIDDEN_DEV_VALUES:
                     self._attributes[key] = value
 
-            await self.async_update_movies()
-            await self.async_update_current_movie()
+            if ATTR_VERSION not in self._attributes:
```

Comment:
We shouldn't add this to the state attributes. It should be be returned from  device_info

Reply:
Fixed